nav2_bt_navigator/src/navigate_to_pose_behavior_tree.cpp

```diff
@@ -71,25 +76,24 @@ BT::NodeStatus NavigateToPoseBehaviorTree::updatePath(BT::TreeNode & tree_node)
   return BT::NodeStatus::RUNNING;
 }
 
-BT::NodeStatus NavigateToPoseBehaviorTree::globalLocalizationServiceRequest()
+BT::NodeStatus
+NavigateToPoseBehaviorTree::globalLocalizationServiceRequest()
 {
   auto request = std::make_shared<std_srvs::srv::Empty::Request>();
   try {
-    auto result = global_localization_.invoke(request, std::chrono::seconds(1));
+    auto result = global_localization_client_->invoke(request, std::chrono::seconds(1));
     return BT::NodeStatus::SUCCESS;
   } catch (std::runtime_error & e) {
     RCLCPP_WARN(node_->get_logger(), e.what());
     return BT::NodeStatus::FAILURE;
   }
 }
 
-BT::NodeStatus NavigateToPoseBehaviorTree::initialPoseReceived(BT::TreeNode & tree_node)
+BT::NodeStatus
+NavigateToPoseBehaviorTree::initialPoseReceived(BT::TreeNode & tree_node)
 {
   auto initPoseReceived = tree_node.blackboard()->template get<bool>("initial_pose_received");
-  if (initPoseReceived) {
-    return BT::NodeStatus::SUCCESS;
-  }
-  return BT::NodeStatus::FAILURE;
+  return initPoseReceived ? BT::NodeStatus::SUCCESS : BT::NodeStatus::FAILURE;
 }
 
```

Comment:
In master there is a new simple action node for clearing the entire costmap.

Reply:
Ok, thanks. We can include that node, if desired, in BTs.